lighthouse-cli/test/smokehouse/a11y/a11y-config.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * Config file for running PWA smokehouse audits for axe.
+ */
+module.exports = {
+  extends: 'lighthouse:default',
+  settings: {
+    onlyCategories: [
+      'accessibility',
+    ],
```

Comment:
since we were talking about it yesterday... is it possible to reuse an existing smoketest?

Reply:
I know we were literally just talking about not adding more yesterday but of all the testers that should probably get their own one, I think a11y is one of them 😆  

maybe we can start by reducing DBW's 6 passes instead? :)